search_api_doc.rst

```diff
@@ -0,0 +1,225 @@
+Search API
+==========
+
+Method::
+   
+   GET http://archive.cnx.org/search?q=
+
+Usage::
+   
+   QUERY [author:AUTHOR] [authorID:ID] [title:TITLE] [subject:SUBJECT] ...
+         [keyword:KEY] [type:TYPE] [language:LANG] [pubYear:YEAR] [sort:SORT]
+
+Brakets denote an optional parameter.
+
+
+Parameters:
+
++---------+---------+---------------------------------+
+| Field   | Type    | Description                     |
++---------+---------+---------------------------------+
+| QUERY   | String  | General search string           |
```

Comment:
``q`` rather than QUERY shouldn't be in the values you are listing here. It is a request parameter rather than a search term/limit parameter.